applicationconfig/client/src/main/java/com/azure/applicationconfig/ApplicationConfigService.java

```diff
@@ -1,23 +1,24 @@
 package com.azure.applicationconfig;
 
-import com.azure.applicationconfig.implementation.PageImpl;
+import com.azure.applicationconfig.implementation.Page;
```

Comment:
Talking about merging terminology, you guys are using `applicationconfig` while in .NET we are using `ApplicationModel.Configuration`
We should evaluate if we can make them the same